services/credential-server-ui/src/components/inputOOBI/GetScannerButton.tsx

```diff
@@ -83,15 +87,28 @@ const GetScannerButton: React.FC<GetScannerButtonProps> = () => {
 
   return (
     <Container sx={{ py: 2 }}>
-      <Box sx={{ display: "flex", justifyContent: "right" }} mb={2}>
+      <Box sx={{ display: "flex", justifyContent: "left" }} mb={1}>
         <Button
-          variant="contained"
-          color="primary"
-          onClick={() => setShowInput(true)}
+          variant="text"
+          onClick={() => onBack()}
         >
-          Get Scanner
+          Back
         </Button>
       </Box>
+      <Typography align="center" mb={2}>
+        Now, to connect the server to your identity wallet, please share an identifier connection QR code. This can be accessed directly in the identifier itself (share button), or in the Connections page by clicking the + icon and “Provide a QR Code"
+      </Typography>
+      {showScannerBtn &&
+        <Box sx={{ display: "flex", justifyContent: "center" }} mb={2}>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => setShowInput(true)}
+          >
+            Show scanner
```

Comment:
Can we remove this blue button and have it automatically show the scanner? (the one that has "Start scanning" and "Scan an image file")